dask/array/reductions.py

```diff
@@ -451,74 +451,74 @@ def nansum(a, axis=None, dtype=None, keepdims=False, split_every=None, out=None)
     )
 
 
-with ignoring(AttributeError):
```

Comment:
Why remove this `ignoring(AttributeError)`? Is this a NumPy version thing we no longer need?

Reply:
Yeah that was my read of it. It was added 6 years ago and seems to work fine with our oldest supported version of numpy.

Reply:
Great, thanks for clarifying 👍